release-notes/opensearch-security-dashboards-plugin.release-notes-2.3.0.0.md

```diff
@@ -1,6 +1,6 @@
-## 2022-09-15 Version 2.3.0.0
+## 2022-10-04 Version 2.3.1.0
```

Comment:
I don't think this change was intented to be added to this PR, can you roll it back?